mne/beamformer/_lcmv.py

```diff
@@ -255,8 +249,14 @@ def _apply_lcmv(data, filters, info, tmin, max_ori_out):
     logger.info('[done]')
 
 
+def _deprecate_max_ori_out(max_ori_out):
+    if max_ori_out is not None:
+        warn('max_ori_out will be removed in 1.0, do not pass it as an '
+             'argument', DeprecationWarning)
+
+
 @verbose
-def apply_lcmv(evoked, filters, max_ori_out='signed', verbose=None):
+def apply_lcmv(evoked, filters, *, max_ori_out=None, verbose=None):
```

Comment:
I am probably being blind, but: why do we need the * operator here?

Reply:
It forces it to be input as a keyword argument only. In other words, this is not allowed:
```
apply_lcmv(evoked, filters, 'signed')
```

Reply:
Aaah, that makes sense of course. Thanks for explaining!